### PR TITLE
Add quotes for the title separation in log messages

### PR DESCRIPTION
**Pull request by xstefank**

### CODE REVIEW

**Comment by xstefank** [comment]
@The-Huginn please review

File: src/main/java/io/xstefank/wildfly/bot/PullRequestRuleProcessor.java [hunk 1]
```diff
@@ -49,14 +49,14 @@ void pullRequestRuleCheck(
         for (WildFlyConfigFile.WildFlyRule rule : wildflyBotConfigFile.wildfly.rules) {
             if (Matcher.notifyRequestReview(pullRequest, rule)) {
                 if (!rule.notify.isEmpty()) {
-                    LOG.infof("Pull Request %s was matched with a rule, containing notify, with the id: %s.",
+                    LOG.infof("Pull Request \"%s\" was matched with a rule, containing notify, with the id: %s.",
```
Comment (The-Huginn) [inline src/main/java/io/xstefank/wildfly/bot/PullRequestRuleProcessor.java:52]: This is forgotten change from custom logger. I think we don't want something like "Pull Request [#123] - Pull Request "title-123"..." but rather "Pull Request [#123] - title "title-123"... "
```suggestion
                    LOG.infof("title \"%s\" - was matched with a rule, containing notify, with the id: %s.",
```
Or we can update logger with new methods or flags specifying what all do we want to print, i.e. only number / number with title etc. Having default only number for example

File: src/main/java/io/xstefank/wildfly/bot/PullRequestRuleProcessor.java [hunk 2]
```diff
@@ -49,14 +49,14 @@ void pullRequestRuleCheck(
...
                             pullRequest.getTitle(), rule.id != null ? rule.id : "N/A");
                     reviewers.addAll(rule.notify);
                 }
                 labels.addAll(rule.labels);
             } else if (Matcher.notifyComment(pullRequest, rule)) {
                 if (!rule.notify.isEmpty()) {
-                    LOG.infof("Pull Request %s was matched with a rule, containing notify, with the id: %s.",
+                    LOG.infof("Pull Request \"%s\" was matched with a rule, containing notify, with the id: %s.",
```
Comment (The-Huginn) [inline src/main/java/io/xstefank/wildfly/bot/PullRequestRuleProcessor.java:59]: same as above

**Review by The-Huginn — APPROVED** [review]
LGTM, we can create separate issue for what I mentioned

File: src/main/java/io/xstefank/wildfly/bot/PullRequestRuleProcessor.java (hunk 1 above)
Comment (xstefank) [reply src/main/java/io/xstefank/wildfly/bot/PullRequestRuleProcessor.java:52]: go for it

**Pull request merged**